doc/misc/npm-config.md

```diff
@@ -879,6 +879,14 @@ If true, output the npm version and exit successfully.
 
 Only relevant when specified explicitly on the command line.
 
+### version-prefix
+
+* Default: `"v"`
+* Type: string
+
+If set, alters the prefix used when tagging a new version when performing a
+version increment using  `npm-version`.
```

Comment:
This needs a note saying that you should _only_ use this property if you _absolutely need to_, due to the hazards of it breaking something completely unrelated elsewhere.